chromatic/rainbows/actions/shift.py

```diff
@@ -12,8 +12,8 @@ def shift(self, velocity=5*u.km/u.s):
     
     new = self._create_copy()
     
-    c = 3e5 * u.km / u.s #speed of light in km/s
-    new_wavelength = new.wavelength * ( 1 / (1 - velocity/c) )
+    lightspeed = c.c.to('km/s') #speed of light in km/s
+    new_wavelength = new.wavelength * ( 1 / (1 - velocity/lightspeed) )
```

Comment:
I think it's better to use the relativistic longitudinal Doppler effect expression from [here](https://en.wikipedia.org/wiki/Relativistic_Doppler_effect#Relativistic_longitudinal_Doppler_effect). It will *deeply* not matter when  $\beta=v/c$ is small, which it always is for stars, but we might as well do it right in case somebody wants to use `chromatic` for a quasar or something!